Packages/com.unity.multiplayer.samples.coop/Utilities/Testing/TestUtilities.cs

```diff
@@ -0,0 +1,146 @@
+using System;
```

Comment:
quick note after conversation, all tests boss room specific need to be moved to boss room. Utilities should only contain utilities tests

Reply:
This should be good to go 👍 . PR 487 will be updated to move around the tests inside Boss Room project instead.